changelog/_unreleased/2025-10-16-repeat-payment-finalize.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Repeat payment finalize
+issue: #12593
+---
+# Core
+* Added `src/Core/Migration/V6_7/Migration1760438732AddConsumedToPaymentToken.php`, which introduces a new consumed column to the payment_token table to track token usage.
```

Comment:
are all those points relevant for externals to know? Remember, that the purpose of a changelog is not to describe every step you made.